src/Sylius/Bundle/ApiBundle/Resources/config/services/filters.xml

```diff
@@ -113,9 +116,10 @@
             <tag name="api_platform.filter" />
         </service>
 
-        <service id="sylius.api.product_image_product_variants_filter" parent="api_platform.doctrine.orm.search_filter" public="true">
+        <service id="sylius_api.search_filter.product_image" parent="api_platform.doctrine.orm.search_filter" public="true">
             <argument type="collection">
                 <argument key="productVariants">exact</argument>
```

Comment:
do we need this filter?

Reply:
Why not? I would leave it, especially as it is tested 🖖🏻